dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/LksgData.kt

```diff
@@ -0,0 +1,221 @@
+import java.math.BigDecimal
+import java.time.LocalDate
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the LKSG framework
+ */
+@DataType("lksg")
+data class LKSGData(
+    val dataDate: LocalDate? = null,
+
+    val LKSGinScope: YesNo? = null,
+
+    val companyLegalForm: String? = null,
+
+    val VATidentificationNumber: String? = null,
+
+    val numberOfEmployees: BigDecimal? = null,
+
+    val shareOfTemporaryWorkers: BigDecimal? = null,
+
+    val totalRevenue: BigDecimal? = null,
+
+    val totalRevenueCurrency: String? = null,
+
+    val responsibilitiesForFairWorkingConditions: YesNo? = null,
+
+    val responsibilitiesForTheEnvironment: YesNo? = null,
+
+    val responsibilitiesForOccupationalSafety: YesNo? = null,
+
+    val riskManagementSystem: YesNo? = null,
+
+    val grievanceHandlingMechanism: YesNo? = null,
+
+    val grievanceHandlingMechanismUsedForReporting: YesNo? = null,
+
+    val codeOfConduct: YesNo? = null,
+
+    val codeOfConductRiskManagementTopics: YesNo? = null,
+
+    val codeOfConductTraining: YesNo? = null,
+
+    val legalProceedings: YesNo? = null,
+
+    val employeeUnder18: YesNo? = null,
+
+    val employeeUnder18Under15: YesNo? = null,
+
+    val employeeUnder18Apprentices: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePrevention: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePreventionEmploymentContracts: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePreventionJobDescription: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePreventionIdentityDocuments: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePreventionTraining: YesNo? = null,
+
+    val employmentUnderLocalMinimumAgePreventionCheckingOfLegalMinimumAge: YesNo? = null,
+
+    val forcedLabourAndSlaveryPrevention: YesNo? = null,
+
+    val forcedLabourAndSlaveryPreventionEmploymentContracts: YesNo? = null,
+
+    val forcedLabourAndSlaveryPreventionIdentityDocuments: YesNo? = null,
+
+    val forcedLabourAndSlaveryPreventionFreeMovement: YesNo? = null,
+
+    val forcedLabourAndSlaveryPreventionProvisionSocialRoomsAndToilets: YesNo? = null,
+
+    val forcedLabourAndSlaveryPreventionProvisionTraining: YesNo? = null,
+
+    val documentedWorkingHoursAndWages: YesNo? = null,
+
+    val adequateLivingWage: YesNo? = null,
+
+    val regularWagesProcessFlow: YesNo? = null,
+
+    val fixedHourlyWages: YesNo? = null,
+
+    val OSHmonitoring: YesNo? = null,
+
+    val OSHpolicy: YesNo? = null,
+
+    val OSHpolicyPersonalProtectiveEquipment: YesNo? = null,
+
+    val OSHpolicyMachineSafety: YesNo? = null,
+
+    val OSHpolicyDisasterBehaviouralResponse: YesNo? = null,
+
+    val OSHpolicyAccidentsBehaviouralResponse: YesNo? = null,
+
+    val OSHpolicyWorkplaceErgonomics: YesNo? = null,
+
+    val OSHpolicyHandlingChemicalsAndOtherHazardousSubstances: YesNo? = null,
+
+    val OSHpolicyFireProtection: YesNo? = null,
+
+    val OSHpolicyWorkingHours: YesNo? = null,
```

Comment:
detekt failing due to naming

Reply:
Seems like Kata fixed this, but the changes got lost during merging remote on her local branch.
So waiting for re-fix.